### DISCUSSION TITLE
arrows acting all fucky in 1.7

### DISCUSSION THREAD

**Discussion by otterola** [KadeArchive/Kade-Engine#2132] [Q&A]
![image](https://user-images.githubusercontent.com/89815386/132073951-fd93df7c-8387-4002-9082-62ecccf5c915.png)
yeah i updated my mod from 1.5.2 to 1.7 and it just. did this for some reason. help me please @KadeDev

**Comment by Riskerman21** [comment]
i don't know about this chief what did you do?
you sure that the porting was smooth ?

**Comment by somefnfplayerlol** [comment]
check the NOTE_assets.xml file, the animation names may not be correct

**Comment by NK-523** [comment]
From the whitty mod,
![NOTE_assets](https://user-images.githubusercontent.com/77185327/132573386-afcd89eb-6e3c-4e0e-be00-fa3344abfef4.png)
from my 1.7 build, (**your .png should look like this**.)
![NOTE_assets](https://user-images.githubusercontent.com/77185327/132573467-0420861e-1d9a-49a8-b7b0-88237dfa163e.png)
